LaCE-onboarding-guide.md

```diff
@@ -53,6 +71,8 @@ This page will help you solve common queries that you may face while working on
 
 ## 4. Creating a Classroom
 
+![config-tab](https://github.com/SanjaySajuJacob/oppia-web-developer-docs/assets/70763993/2f73ea74-d2ff-4e09-a962-be049bfcb442)
```

Comment:
This is not needed as it will be same for rows from 77 to 83

Reply:
Hi @prafulbbandre , I this section is being worked on in your PR as of now. Thanks!